torch/csrc/jit/register_special_ops.cpp

```diff
@@ -150,6 +150,68 @@ void recursiveStore(
   }
 }
 
+template<bool if_set_requires_grad>
+Operation createTensorFromList(const Node* node) {
+  auto input = node->inputs().at(0);
+  auto elem_type = input->type();
+  while (auto list_type = elem_type->cast<ListType>()) {
```

Comment:
This doesn't work for as_tensor because `checkListInputType` expects a non-tensor input.